tasks/sync_repos.py

```diff
@@ -89,7 +89,7 @@ async def sync_repos_using_integration(self, db_session, git, ownerid, username)
         with metrics.timer(f"{metrics_scope}.sync_repos_using_integration.list_repos"):
             repos = await git.list_repos_using_installation(username)
         if repos:
-            service_ids = {repo["id"] for repo in repos}
+            service_ids = {str(repo["repo"]["service_id"]) for repo in repos}
```

Comment:
why don't we just make this a string in the return format for both this and `list_repos()`? i don't think we use them anywhere else

Reply:
Made this change in shared and updated things here accordingly